content/en/getting_started/synthetics/private_location.md

```diff
@@ -41,48 +43,50 @@ The private location worker is available on Docker Hub:
 
 1. Set up a [Vagrant Ubuntu 16.04 virtual machine][2].
 2. Install [Docker][4] on that machine.
-3. In the Datadog app, hover over **[UX Monitoring][5]** and select *Settings* -> *Private Locations*. Click **Add Private Location**.
-4. Fill out your private location details (only `Name` and `API key` fields are mandatory). Click **Save Location and Generate Configuration File** to generate the configuration file associated with your private location on your worker.
-5. Specify the proxy URL if the traffic between your private location and Datadog needs to go through a proxy. You can also optionally toggle the **Block reserved IPs** button to block a default set of reserved IP ranges ([IPv4 address registry][6] and [IPv6 address registry][7]).
-6. Copy and paste your private location configuration file to your working directory.
-
-    **Note**: The configuration file contains secrets for private location authentication, test configuration decryption, and test result encryption. Datadog does not store the secrets, so store them locally before leaving the Private Locations screen. **You need to be able to reference these secrets again if you decide to add more workers, or to install workers on another host.**
+3. In the Datadog application, hover over **[UX Monitoring][5]** and select **Settings** > **Private Locations**. 
+4. Click **Add Private Location**.
+5. Fill out your private location details (only `Name` and `API key` fields are mandatory). 
```

Comment:
Let's add something like "Depending on where you installed your private location (e.g., behind a proxy, etc.), you might have some additional parameters to configure" with a link to https://docs.datadoghq.com/synthetics/private_locations/configuration/#configuration-options